app/components/footer.tsx

```diff
@@ -1,5 +1,7 @@
-import React from "react";
 import { Link } from "@remix-run/react";
+import styles from '../styles/index.css';
+
+export const links = () => [{ rel: "stylesheet", href: styles }];
```

Comment:
That said the main index of styles is imported in the root.tsx file so you shouldn't have to add them here.